developers/contributing_how_tos.md

```diff
@@ -0,0 +1,63 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.0
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How to contribute a napari How-To
+
+This should be a sentence (or brief paragraph) about what the reader will achieve after following this
```

Comment:
I think this phrasing is a bit too meta, like it's conflating this guide with a template in itself.
It reads more like "we meant to put content here at this placeholder but forgot", rather than the message you want to convey.